src/app/proposals/draft/[id]/page.tsx

```diff
@@ -74,14 +74,14 @@ export default async function DraftProposalPage({
           </section>
           <section className="col-span-1">
             <div className="bg-wash border border-line rounded-2xl p-4">
-              <p className="mt-2">
+              <div className="mt-2">
```

Comment:
React was complaining about nesting a dev in a p tag... simple to change, doesn't matter much